packages/salesforcedx-utils-vscode/src/config/configUtil.ts

```diff
@@ -54,11 +55,26 @@ export class ConfigUtil {
   }
 
   public static async getDefaultUsernameOrAlias(): Promise<string | undefined> {
-    const configAggregator = await ConfigAggregatorProvider.getInstance().getConfigAggregator();
-    const defaultUsernameOrAlias = configAggregator.getPropertyValue(
-      OrgConfigProperties.TARGET_ORG
-    );
-    return defaultUsernameOrAlias ? String(defaultUsernameOrAlias) : undefined;
+    try {
+      const configAggregator = await ConfigAggregatorProvider.getInstance().getConfigAggregator();
+      const defaultUsernameOrAlias = configAggregator.getPropertyValue(
+        OrgConfigProperties.TARGET_ORG
+      );
+      if (defaultUsernameOrAlias === undefined) {
+        return undefined;
+      }
+
+      return JSON.stringify(defaultUsernameOrAlias).replace(/\"/g, '');
```

Comment:
I'm curious why we're doing this (and also, would `String(defaultUsernameOrAlias)` be nicer? What types can be returned from `configAggregator.getPropertyValue()`?

Reply:
I think the issue with `String(defaultUsernameOrAlias)` is that if `defaultUsernameOrAlias` is undefined, it returns back `"undefined"` and not `undefined`.